cdap-docs/hydrator-manual/source/creating-pipelines.rst

```diff
@@ -142,8 +141,8 @@ To use Hydrator Studio to create a batch pipeline:
   any of the :ref:`batch source plugins <cask-hydrator-plugins-batch-sources>` or the
   :ref:`batch sink plugins <cask-hydrator-plugins-batch-sinks>`.
 
-- Typically, you will need at a minimum a source, a sink, and any optional transformations
-  that are needed being the source and sink stages.
+- Typically, you will need at a minimum a source, a sink, and any optional transformations or analytics
+  that are needed between the source and sink stages.
```

Comment:
Perhaps remove the repeated *need*?:

that are *required* between the source and sink stages.